BREAKING.md

```diff
@@ -152,6 +153,15 @@ Note: `IConnectableRuntime` is only to be implemented internally, so removing th
 
 `IDocumentServiceFactory.protocolName` was deprecated in 2.0.0-internal.3.0.0 and has now been removed.
 
+### Closing Container no longer disposes
+
+Calling `IContainer.close(...)` will no longer dispose the container runtime, document service, or document storage service.
+
+If the container is not expected to be used after the `close(...)` call, replace it instead with a `IContainer.dispose(...)` call. Using `IContainer.dispose(...)` will no longer switch the container to "readonly" mode and relevant code should instead listen to the Container's "disposed" event.
+Otherwise, to retain all previous behavior, add a call to `IContainer.dispose(...)` after every `close(...)` call (passing the same error object if present).
```

Comment:
Seems like `close` then `dispose` is not what anyone should actually do.  They should choose whether they want `close` semantics or `dispose` semantics.  Right?

Reply:
@kian-thompson - How about "As a short-term measure to retain all previous behavior"? And then follow-up with "
```suggestion
As a short-term measure to retain all previous behavior, add a call to `IContainer.dispose(...)` after every `close(...)` call (passing the same error object if present). However, in the long run it's better to wait to call `dispose` until your application is shutting down.
```

@vladsud do you even agree with how I phrased it?  I'm struggling to understand the semantics of `dispose` v. `close-dispose` v, `close...dispose`.  How critical is it for the app to _ever_ call `dispose`?

And in cases where internally we are invoking `close`, I'm not sure we have enough info to decide about `close` v. `dispose`.  Should we default to `close` only, and trust that the application will call `dispose` from their `close` handler if they're all done?

Reply:
I'd personally prefer for apps to reassess their behavior right away. I.e. if the intention of particular callsite is to get rid of container and never use it again, they should call only dispose(). If the intent is to leave container on the screen in fully disconnected and detached from storage state, then they should call close().

Given that we are "breaking" it, I'd also look at how close() is used today on Loop side, and possible remove ability for external code (i.e. calling public API) to provide us error object on close().